VAMobile/src/utils/errors.ts

```diff
@@ -44,6 +44,10 @@ export const getCommonErrorFromAPIError = (error: APIError, screenID?: ScreenIDT
     return CommonErrorTypesConstants.APP_LEVEL_ERROR_DISABILITY_RATING
   } else if (screenID === ScreenIDTypesConstants.APPOINTMENTS_SCREEN_ID && error.status && error.status >= 500) {
     return CommonErrorTypesConstants.APP_LEVEL_ERROR_APPOINTMENTS
+  } else if (!error.json?.errors[0].refreshable && error.status && error.status === 418) {
+    return CommonErrorTypesConstants.CUSTOM_ERROR
+  } else if (error.json?.errors[0].refreshable && error.status && error.status === 418) {
+    return CommonErrorTypesConstants.CUSTOM_ERROR_WITH_REFRESH
```

Comment:
```suggestion
```

Reply:
I may have mucked up the syntax playing in the web browser but something like this might be a little cleaner.

Reply:
got it!